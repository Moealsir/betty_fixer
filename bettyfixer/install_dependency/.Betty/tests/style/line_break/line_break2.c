#include <stdio.h>

/**
 * main - Entry point
 *
 * Return: Always 0 (Success)
 */
int main(void)
{
	printf("hello"); /* a =
	12; return a; */
	printf("hello"); /* a = 12; return a;*/
	printf("hello"); a = 12; return a;
}

int func(void)
{
	int i = 0;

	if (i != 3)
	{
		_putchar('E'); _putchar('r'); _putchar('r'); _putchar('o');
		_putchar('r'); _putchar('\n'); exit(98);
	}

	return (0);
}
