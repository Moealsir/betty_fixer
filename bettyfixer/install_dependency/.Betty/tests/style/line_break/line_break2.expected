tests/style/line_break/line_break2.c:11: WARNING: Block comments use * on subsequent lines
tests/style/line_break/line_break2.c:11: WARNING: Block comments use a trailing */ on a separate line
tests/style/line_break/line_break2.c:13: WARNING: Multiple instructions on a single line is forbidden
tests/style/line_break/line_break2.c:22: WARNING: Multiple instructions on a single line is forbidden
tests/style/line_break/line_break2.c:23: WARNING: Multiple instructions on a single line is forbidden
total: 0 errors, 5 warnings, 27 lines checked
